config/control_ur5.yaml

```diff
@@ -0,0 +1,29 @@
+arm_controller:
```

Comment:
I thought of doing that, but it broke a bunch of other things. Given the current timeline we have.. I don't have time to poke around and I'd rather get it working on the real robot before making enhancements, but feel free to change it if you have time.